mw-webapp/src/component/inProgress/InProgress.module.scss

```diff
@@ -10,6 +10,6 @@
 
 @keyframes progress {
   100% {
-      width: 100%;
+    width: 100%;
   }
 }
```

Comment:
PLs create task about adjust lint, prettier for scss files

Reply:
I think Prettier did this

Reply:
UPD: Yes it is Prettier